src/core/js/a11y/browserFocus.js

```diff
@@ -0,0 +1,84 @@
+define([
+    'core/js/adapt'
+], function(Adapt) {
+
+    /**
+     * Browser modifications to focus handling.
+     * @class
+     */
+    var BrowserFocus = Backbone.Controller.extend({
+
+        initialize: function() {
+            _.bindAll(this, '_onBlur', '_onClick');
+            this.$body = $('body');
+            this.listenTo(Adapt, {
+                'accessibility:ready': this._attachEventListeners
+            });
+        },
+
+        _attachEventListeners: function() {
+            this.$body
+                .on('blur', '*', this._onBlur)
+                .on('blur', this._onBlur);
+            // 'Capture' event attachment for click
+            this.$body[0].addEventListener('click', this._onClick, true);
+        },
+
+        /**
+         * When any element in the document receives a blur event,
+         * check to see if it needs the `data-a11y-force-focus` attribute removing
+         * and check to see if it was blurred because a disabled attribute was added.
+         * If a disabled attribute was added, the focus will be moved forward.
+         *
+         * @param {Object} event jQuery event object.
+         */
+        _onBlur: function(event) {
+            var config = Adapt.a11y.config;
+            if (!config._isEnabled || !config._isFocusNextOnDisabled) {
+                return
+            }
+            var $element = $(event.target);
+            if ($element.is('[data-a11y-force-focus]')) {
+                _.defer(function() {
+                    $element.removeAttr('tabindex data-a11y-force-focus');
+                });
+            }
+            // From here, only check source elements
+            if (event.target !== event.currentTarget) {
+                return;
+            }
+            // Check if element losing focus is losing focus
+            // due to the addition of a disabled class
+            if (!$element.is('[disabled]')) {
+                return;
+            }
+            // Move focus to next readable element
+            Adapt.a11y.focusNext($element);
+        },
+
+        /**
+         * Force focus when clicked on a tabbale element,
+         * making sure `document.activeElement` is updated.
+         *
+         * @param {Object} event jQuery event object.
```

Comment:
If you add `"@types/jquery": "^3.3.31"` to package.json then you can change this to
```suggestion
         * @param {JQuery.Event}
```
and get proper type definitions appearing:
![image](https://user-images.githubusercontent.com/1249597/62935959-678a5400-bdc0-11e9-9baa-c4fb17abf926.png)

Reply:
done